engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/output/table/html/helper/HtmlTextExtractorHelper.java

```diff
@@ -544,6 +555,28 @@ private boolean tryHandleDrawable( final ReportAttributeMap attrs, final long wi
     return true;
   }
 
+  private void convertImageDataToBase64( String name, AttributeList attrList, final ImageContainer image, final String type,
+    final float quality ) throws ContentIOException {
+    int indexStart = name.lastIndexOf( "picture" );
+    String imgAltText = name.substring( indexStart );
+    attrList.setAttribute( HtmlPrinter.XHTML_NAMESPACE, ALT_ATTR, imgAltText );
+    attrList.setAttribute( HtmlPrinter.XHTML_NAMESPACE, SRC_ATTR, "poc-src-placeholder-" + imgAltText );
```

Comment:
you need to remove this line it clearly has "poc" "proof of concept" in it.

Reply:
addressed